Philips.CodeAnalysis.MaintainabilityAnalyzers/Documentation/CopyrightPresentAnalyzer.cs

```diff
@@ -55,32 +56,25 @@ private void Analyze(SyntaxNodeAnalysisContext context)
 				return;
 			}
 
-			var first = node.GetLeadingTrivia();
+			var location = GetSquiggleLocation(node.SyntaxTree);
```

Comment:
This call can be postponed until a diagnostic is triggered

Reply:
Correct but I opted for here to simplify the code. (The first diagnostic could be 4 lines later.) It's a performant call not involving the semantic model.